inc/HTTP/HTTPClient.php

```diff
@@ -319,8 +319,7 @@ public function sendRequest($url, $data = '', $method = 'GET')
                 if ($match[1] > $this->max_bodysize) {
                     if ($this->max_bodysize_abort)
                         throw new HTTPClientException('Reported content length exceeds allowed response size');
-                    else
-                        $this->error = 'Reported content length exceeds allowed response size';
+                    else $this->error = 'Reported content length exceeds allowed response size';
```

Comment:
Is there a rule that add to these one line statements brackets as well? such as line 321. It should not add brackets to one-liners that have their statement on the same line as the if or else. That is better to do manually.

Reply:
Unfortunately I couldn't find anything in either rector nor codesniffer. I think the latter would probably the tool that should provide it